orchagent/portsorch.h

```diff
@@ -88,20 +88,22 @@ class PortsOrch : public Orch, public Subject
     bool addHostIntfs(sai_object_id_t router_intfs_id, string alias, sai_object_id_t &host_intfs_id);
 
     bool addBridgePort(Port &port);
-    bool removeBridgePort(Port port);
+    bool removeBridgePort(Port &port);
 
     bool addVlan(string vlan);
     bool removeVlan(Port vlan);
-    bool addVlanMember(Port vlan, Port port, string& tagging_mode);
-    bool removeVlanMember(Port vlan, Port port);
+    bool addVlanMember(Port &vlan, Port &port, string& tagging_mode);
+    bool removeVlanMember(Port &vlan, Port &port);
 
     bool addLag(string lag);
     bool removeLag(Port lag);
-    bool addLagMember(Port lag, Port port);
-    bool removeLagMember(Port lag, Port port);
-
+    bool addLagMember(Port &lag, Port &port);
+    bool removeLagMember(Port &lag, Port &port);
+    void getLagMember(Port &lag, vector<Port> &portv);
     bool setPortAdminStatus(sai_object_id_t id, bool up);
     bool setPortMtu(sai_object_id_t id, sai_uint32_t mtu);
+    bool setPortPvid (Port &port, sai_uint32_t pvid);
```

Comment:
>setPortPvid [](start = 9, length = 11)

It is better to rename it as setPvid, no need to give a port. actually, I think we should also remove the Port from the functions names the setPortAdminStatus/setPortMtu/setPortSpeed/getPortSpeed